components/desktop/mate/mate-session-manager/Makefile

```diff
@@ -52,3 +56,4 @@ REQUIRED_PACKAGES += x11/library/libx11
 REQUIRED_PACKAGES += x11/library/libxext
 REQUIRED_PACKAGES += x11/library/libxrender
 REQUIRED_PACKAGES += x11/library/libxtst
+REQUIRED_PACKAGES += x11/library/mesa
```

Comment:
When I run gmake REQUIRED_PACKAGES x11/library/mesa will be removed here.
So maybe it's a good idea to add a comment or move the dependency out of the generated area into a separate one.

Reply:
It got removed because of [this](https://github.com/OpenIndiana/oi-userland/pull/16891#discussion_r1564934797).